src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+body {
+  box-sizing: border-box;
+  font-family: Avenir, Times, serif;
+}
+
+.block_input {
+  position: relative;
+  margin: 20px 0;
+}
+
+.icon_search {
+  position: absolute;
```

Comment:
there is no need to move position separately from other styles, it is ok if position is written several times